ros2doctor/ros2doctor/api/__init__.py

```diff
@@ -15,19 +15,68 @@
 from pkg_resources import iter_entry_points
 
 
+class DoctorCheck:
+    """
+    Abstract based class of ros2doctor check modules.
+
+    category method returns a string that includes the module name.
```

Comment:
This should say what the returned string is for: for linking reports and checks.